app/assets/javascripts/order-form.js

```diff
@@ -92,6 +92,6 @@ $(function () {
 
     // 合計金額の計算を行う
     $("table tr").find(".order-form-money")
```

Comment:
tdタグにidを指定してjQueryで取得するのが確実です。

まずtdタグにidを指定する。
```
      %table.table.table-hover
        %tbody
          %tr
            %th{scope: "row"}
              時給
            %td#hourly_pay
              1000円
          %tr
            %th{scope: "row"}
              オプション金額
            %td#option_ money
              1000円
          %tr
            %th{scope: "row"}
              手数料
            %td#fee
              300円
            %th
              合計金額
            %td#amount
              2300円
```


jsは以下のようにする。
```suggestion
　// これでオプション金額が取れます。他も同じ感じで取る。
　// 1000円
    $("#option_ money").text()
```